.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "esbonio.sphinx.confDir": ""
+}
```

Comment:
This is from your directory of VS Code settings and shouldn't be committed. You should be able to add the .vscode directory to your gitignore file

Reply:
Should I remove this from the PR?